DevProject/Assets/csc.rsp

```diff
@@ -0,0 +1,2 @@
+-warnaserror+
```

Comment:
This is what upm-ci uses:
https://github.cds.internal.unity3d.com/unity/upm-ci-utils/blob/09fc1f2d4ec75e46097bbe236867abd40232bd63/lib/project_utils.ts#L443

Warning [618](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0618) is using [Obsolete] - we can remove this if we want to make this a failure.